repair/row_level.cc

```diff
@@ -552,12 +552,12 @@ void flush_rows(schema_ptr s, std::list<repair_row>& rows, lw_shared_ptr<repair_
     auto cmp = position_in_partition::tri_compare(*s);
     lw_shared_ptr<mutation_fragment> last_mf;
     lw_shared_ptr<const decorated_key_with_hash> last_dk;
+    writer->start_streaming(db, sys_dist_ks, view_update_generator);
     for (auto& r : rows) {
         thread::maybe_yield();
         if (!r.dirty_on_master()) {
             continue;
         }
-        writer->create_writer(db, sys_dist_ks, view_update_generator);
```

Comment:
perhaps you should move it in a separate patch explaining the motivation.

Reply:
Yes. This changes the logic of the code. Before if the rows are empty, the writer is not created. Now, it creates the write unconditionally.

Reply:
Let's not do it for now in this series. The optimization is minor.

Reply:
No strong opinion here. I didn't understand the reasoning of calling create_writer multiple times in the loop, while all, apart from the first one, do early exit because the writer was already created. The unit/dtests didn't show any difference. It wasn't intended as performance optimisation. I can definitely remove it.